app/services/katello/pulp3/content_view_version/export.rb

```diff
@@ -55,8 +55,8 @@ def generate_exporter_path
           return base_path if base_path
           export_path = "#{content_view_version.content_view}/#{content_view_version.version}/"
```

Comment:
`#{content_view_version.content_view} `evaluates to name but it is also rails magic..would make it clearer to use label to avoid whitespaces etc..

Reply:
Good suggestion. Also gsubing at the end is a bad idea, because it may underscore  `Setting['pulpcore_export_destination']`  which means it will error out if that directory had a space in it.

Reply:
updated